.eslintrc.js

```diff
@@ -1,9 +1,11 @@
 module.exports = {
   extends: ['@mate-academy/eslint-config-react-typescript', 'plugin:cypress/recommended'],
   rules: {
+    "react/require-default-props": "off",
     'max-len': ['error', {
       ignoreTemplateLiterals: true,
       ignoreComments: true,
+      ignorePattern: "<path([\\s\\S]*?)/>",
```

Comment:
Consider adding a comment explaining why `<path>` should be ignored by `max-len` rule. 🤔